test/lib/mayaUsd/nodes/testProxyShapeBase.py

```diff
@@ -57,8 +57,8 @@ def testDuplicateProxyStageAnonymous(self):
 
         # create a proxy shape and add a Capsule prim
         import mayaUsd_createStageWithNewLayer
-        mayaUsd_createStageWithNewLayer.createStageWithNewLayer()
-        proxyShapePath = ufe.PathString.path('|stage1|stageShape1')
+        proxyShape = mayaUsd_createStageWithNewLayer.createStageWithNewLayer()
+        proxyShapePath = ufe.PathString.path(proxyShape)
```

Comment:
Modified all tests that use either of these two helpers (above) to use the new return value.